visidata/_open.py

```diff
@@ -185,7 +185,9 @@ def loadInternalSheet(vd, cls, p, **kwargs):
 
 BaseSheet.addCommand('o', 'open-file', 'vd.push(openSource(inputFilename("open: "), create=True))', 'Open file or URL')
 TableSheet.addCommand('zo', 'open-cell-file', 'vd.push(openSource(cursorDisplay) or fail(f"file {cursorDisplay} does not exist"))', 'Open file or URL from path in current cell')
+BaseSheet.addCommand('zq', 'reopen-last-closed', 'push(allSheets[-1])', 'reopen most recently closed sheet')
```

Comment:
Does `zq` feel good to you at all??  It makes me feel uneasy to press `q` when I want to do the reverse of quit.

Reply:
Yeeeea, I would only ever use the long command in fear of miss typing the z

Reply:
not strictly, but it feels like we want something that makes quitting feel less stressful, like Shift+U does for undo.  I'm kind of leaning towards `g Shift+U` at the moment, since otherwise that would feel like "undo everything" which is just a reload.  Also I think the longname should be `open-last-closed` or maybe `undo-last-quit`, instead of `reopen` which would be a new verb.

Reply:
My vote is`undo-last-quit` 👍

Reply:
`zq` is possibly one of the the most awkward to type key bindings known to man. It doesn't bother me because this really is a "oh shit" command rather than a command that's part of my standard workflow. Logically I think that `zq` makes more sense than `g Shift+U`, but I don't feel strongly about this, so I changed it now to `g Shift+U`.

I disagree about the function name. I've taken the "reopen" teminology from Chrome. The command `undo-last-quit` is awkward enough, but having to change the menu item to "Undo last quit" would be even weirder.

Reply:
The longname and the menu text don't have to be the same.